Altis_Life.Altis/core/civilian/fn_civMarkers.sqf

```diff
@@ -10,32 +10,38 @@ private _members = [];
 
 for "_i" from 0 to 1 step 0 do {
     sleep 0.5;
-    if (visibleMap) then
-    {
+    if (visibleMap) then {
         _members = units (group player);
         {
             if !(_x isEqualTo player) then {
-                _marker = createMarkerLocal [format ["%1_marker",_x],visiblePosition _x];
+                private _marker = createMarkerLocal [format ["%1_marker",_x],visiblePosition _x];
                 _marker setMarkerColorLocal "ColorCivilian";
                 _marker setMarkerTypeLocal "Mil_dot";
                 _marker setMarkerTextLocal format ["%1", _x getVariable ["realname",name _x]];
                 _markers pushBack [_marker,_x];
             };
-        } forEach _members;
+        } count _members;
 
         while {visibleMap} do
         {
             {
-                private _unit = _x select 1;
+                _x params [
+                    ["_mark","",[""]],
```

Comment:
Is this really a string?

Reply:
"MarkerOne" setMarkerPosLocal getMarkerPos "MarkerTwo";